packages/container/lib/container.js

```diff
@@ -28,6 +39,9 @@ export default function Container(registry, options) {
   this.isDestroyed = false;
 }
 
+Container.__FACTORY_FOR__ = FACTORY_FOR;
```

Comment:
This seems bad but don't know how to make the Node tests pass without doing this.